### DISCUSSION TITLE
Language selection/setting

### DISCUSSION THREAD

**Discussion by freddepettersson** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#20] [Q&A] [answered]
Just a idea
I know that the code select language due to the settings in you computer .
But i would be great to also have the possibilly to  ex. set the language to english during testing etc

/Fredrik

**Comment by fjallemark** [comment, marked as answer]
Language is selected by settings in your browser, not in your computer in general. Often you can select which language you prefer to read web pages in, which is not the same as the browser user interface language. So it is quite easy to set a new request language. You can also use different language settings in different browsers.
